tools/codegen/api/lazy.py

```diff
@@ -57,6 +57,9 @@ def process_ir_type(typ: Type) -> Union[BaseCType, VectorCType, OptionalCType, L
         if str(typ.elem) == 'Tensor?':
             # TODO(whc) is this actually correct? or should it use a Vector like above
             return ListCType(OptionalCType(BaseCType(valueT)))
```

Comment:
A silly q. why are we remapping `ListCType(BaseTy.Tensor)` to `tensorListValueT`?
To generate the cpp type declarations e.g.  `tensorListValueT = BaseCppType('torch::lazy', 'Value')` or does it just look cleaner this way?

Reply:
i'm not sure i understand the question.. sorry.

I also don't have a lot of confidence that I did this in the most optimal way.  maybe @bdhirsh has a suggestion.